packages/dotlottie-js/src/tests/lottie-image-node.spec.ts

```diff
@@ -210,13 +213,70 @@ describe('LottieImage', () => {
         expect(uniqueImages.length).toBe(5);
 
         expect(uniqueImages.map((image) => image.fileName)).toEqual([
-          'image_1.png',
-          'image_2.png',
+          'image_1.jpeg',
+          'image_2.jpeg',
           'image_4.png',
           'image_5.png',
           'image_9.png',
         ]);
         expect(uniqueImages.map((image) => image.id)).toEqual(['image_1', 'image_2', 'image_4', 'image_5', 'image_9']);
       });
   });
+
+  it('Properly detects mimetype of images.', async () => {
```

Comment:
[suggestion]
Add unit tests for `getMimeTypeFromBase64` to cover the logic for detecting all supported MIME types. The responsibility for preventing similar issues in the future lies with `getMimeTypeFromBase64`.

Since the logic for inlining assets is already covered by other tests, I don't see the value in adding more integration tests to verify if `getMimeTypeFromBase64` is functioning as expected.

Reply:
added test for all supported file type @theashraf